### ISSUE TITLE
Add subreddits from reddit clones

### ISSUE THREAD

**Issue by nitrocode** [ccrama/Slide#3302]
Since new reddit clones have been popping up like [teddit.net](https://teddit.net), would maintainers of slide consider the ability to add subreddits from reddit clones?

**Comment by Aq32** [comment]
I would probably go a step further and ask if it's possible to allow an option in settings for users so that all connections to teddit be used instead of reddit directly as a move towards respecting user privacy.

**Comment by heretoenhance** [comment]
I totally agree with @Aq32, having the option to extract contents from teddit instead of reddit would be awesome.

**Comment by momobobe** [comment]
Libreddit is another option. Also this is not only for the sake of privacy, but also help people bypass censorship in certain region, in the name of free speech.
